### PR TITLE
[GRO-423] Databricks documentation

### PR DESCRIPTION
**Pull request by elagulsen**
⚠️Not to be merged until Databricks feature flag is turned on 🚧

### CODE REVIEW

File: pages/docs/database-connections/databricks.mdx [hunk 1]
```diff
@@ -0,0 +1,40 @@
+---
+description:
```
Comment (vndrewlee) [inline pages/docs/database-connections/databricks.mdx:2]: please add a description here, important for SEO

File: pages/docs/database-connections/databricks.mdx [hunk 2]
```diff
@@ -0,0 +1,42 @@
+---
+description: Databricks is a flexible data analytics platform with a wide variety of built-in services
+---
+
+# Databricks
+
+[Databricks](https://www.databricks.com) is a data analytics platform built on top of Apache Spark. 
+
+Databricks allows you to combine the robustness of a data warehouse with the flexibility of a data lake via their 
+[Lakehouse architecture](https://www.databricks.com/product/data-lakehouse).
```
Comment (glean-calder) [inline pages/docs/database-connections/databricks.mdx:10]: Nit: they don't capitalize `lakehouse`

File: pages/docs/database-connections/databricks.mdx [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+3. Generate a [personal access token](https://docs.databricks.com/en/dev-tools/auth/pat.html#pat-user) in Databricks for a user with access to the data you want to query:
+    * In the top right corner of your Databricks workspace, click the user icon and navigate to "User Settings".
+    * In the left pane, click into the "Developer" controls. Click the "Manage" button in the "Access tokens" section.
+    * Click "Generate New Token". You will be asked to configure the token's lifespan. Finally, click "Generate".
+    * Copy the token and save it somewhere safe. You will not be able to access it again.
+4. In the sidebar of your workspace, click on "SQL Warehouses" under the "SQL" section. 
+You will see a list of your warehouses. Click on the name of the warehouse you want to connect to.
+Then, navigating to the "Connection details" tab, you can find the hostname and HTTP path of your warehouse.
+5. In Hashboard, go to the [Data sources](https://hashboard.com/app/p/data-sources) page.
+6. Click `+ Add connection`, select `Databricks` and fill out the fields below.
```
Comment (glean-calder) [inline pages/docs/database-connections/databricks.mdx:32]: Would be valuable to normalize how we cite buttons as either `text` or "text" within this section. I think we do indeed usually go with `text`.

File: pages/docs/database-connections/databricks.mdx [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+### Settings
+
+- **Connection Name:** A nickname for your connection. Not used to connect to your database.
```
Comment (glean-calder) [inline pages/docs/database-connections/databricks.mdx:36]:
```suggestion
- **Connection name:** A nickname for your connection. Not used to connect to your database.
```

**Review by glean-calder — APPROVED** [review]
Just nits, lgtm!

**Pull request merged**